Assets/Scripts/ResultManager.cs

```diff
@@ -0,0 +1,78 @@
+using System;
+using System.Collections.Generic;
+using UnityEngine;
+using TMPro;
+
+public class ResultManager : MonoBehaviour
+{
+    [SerializeField] private TextMeshProUGUI scoreText;
+    [SerializeField] private TextMeshProUGUI resultMessage;
+    [SerializeField] private GameObject victoryParticlePrefab;
+    [SerializeField] private Canvas canvas;
```

Comment:
一応命名ルールを統一させておきたいから、privateなフィールドには頭にアンダーバーを付けておこう。

```suggestion
    [SerializeField] private TextMeshProUGUI _scoreText;
    [SerializeField] private TextMeshProUGUI _resultMessage;
    [SerializeField] private GameObject _victoryParticlePrefab;
    [SerializeField] private Canvas _canvas;
```

Reply:
すんません、めっちゃ忘れます😅